ci-operator/config/openshift/cloud-network-config-controller/openshift-cloud-network-config-controller-master.yaml

```diff
@@ -3,13 +3,25 @@ base_images:
     name: "4.8"
     namespace: ocp
     tag: base
+  ocp_builder_rhel-8-golang-1.15-openshift-4.8:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.8
 binary_build_commands: make build
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
     tag: golang-1.15
 canonical_go_repository: openshift/cloud-network-config-controller
+images:
+- dockerfile_path: Dockerfile
+  from: base
+  inputs:
+    ocp_builder_rhel-8-golang-1.15-openshift-4.8:
+      as:
+      - registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.8
```

Comment:
Why not using this directly in the dockerfile?

Reply:
It is used in the Dockerfile, see: https://github.com/openshift/cloud-network-config-controller/blob/master/Dockerfile#L1 

But honestly, I don't know how this really works. But this is defined in all repositories' config files, so I just do what other people do :sheep:

Reply:
This stanza means: The image `ocp_builder_rhel-8-golang-1.15-openshift` (a name from `base_images`) will be used instead of `registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.8` references in the Dockerfile. It makes the build use the cluster-local image. It is usually used to replace the "upstream builder images" with OCP builder images in the context of OpenShift CI.

Reply:
See https://docs.ci.openshift.org/docs/architecture/ci-operator/#building-container-images